.github/workflows/release.yml

```diff
@@ -69,3 +86,15 @@ jobs:
         RCLONE_CONFIG_EVESHIPFIT_ACCESS_KEY_ID: ${{ secrets.R2_ACCESS_KEY_ID }}
         RCLONE_CONFIG_EVESHIPFIT_ENDPOINT: ${{ secrets.R2_ENDPOINT }}
         RCLONE_CONFIG_EVESHIPFIT_SECRET_ACCESS_KEY: ${{ secrets.R2_SECRET_ACCESS_KEY }}
+
+    # Build the package after pushing to R2 so that we don't push the javascript files to R2.
+    - name: Build package
+      run: |
+        npm run build
```

Comment:
I get the reasoning, but it also means building can fail, while the package is already deployed on R2. That is not ideal. So I rather have a JSON file too much ending up on R2, than having these kind of build failures. So just move this above the copying to R2 :)